doc/manual/src/command-ref/nix-store/realise.md

```diff
@@ -8,33 +8,36 @@
 
 # Description
 
-The operation `--realise` essentially “builds” the specified store
-paths. Realisation is a somewhat overloaded term:
-
-  - If the store path is a *derivation*, realisation ensures that the
-    output paths of the derivation are [valid] (i.e.,
-    the output path and its closure exist in the file system). This
-    can be done in several ways. First, it is possible that the
-    outputs are already valid, in which case we are done
-    immediately. Otherwise, there may be [substitutes]
-    that produce the outputs (e.g., by downloading them). Finally, the
-    outputs can be produced by running the build task described
-    by the derivation.
-
-  - If the store path is not a derivation, realisation ensures that the
-    specified path is valid (i.e., it and its closure exist in the file
-    system). If the path is already valid, we are done immediately.
-    Otherwise, the path and any missing paths in its closure may be
-    produced through substitutes. If there are no (successful)
-    substitutes, realisation fails.
 
+Each of *paths* is processed as follows:
+
+- If the path is already [valid], do nothing.
+- If the path leads to a [store derivation]:
+  1. Realise the store derivation file itself, as a regular store path.
```

Comment:
```suggestion
- If the path leads to a [store derivation]:
  1. If it is not [valid], substitute the store derivation file itself.
```

If it leads to a `.drv` which is valid, it won't do nothing. "Do nothing" suggests ending the process immediately.